### PR TITLE
Report private rooms separately in /query

### PR DESCRIPTION
**Pull request by sirDonovan**
To eliminate confusion, don't report private chatrooms where the user is
a player, such as in a tournament

### CODE REVIEW

**Comment by Zarel** [comment]
We could alternatively report those separately? "Private games" or something.

**Comment by sirDonovan** [comment]
That sounds fine, but if we're going to keep chatrooms, can we expand it to report all private rooms both users are in?

**Comment by Zarel** [comment]
Sure.

File: commands.js [hunk 1]
```diff
@@ -2910,8 +2910,11 @@ exports.commands = {
 				if (i === 'global') continue;
 				let targetRoom = Rooms.get(i);
 				if (!targetRoom) continue; // shouldn't happen
-				if (targetRoom.isPrivate && !(i in user.games)) continue;
 				let roomData = {};
+				if (targetRoom.isPrivate) {
+					if (!(i in user.roomCount)) continue;
+					roomData.isPrivate = true;
+				}
```
Comment (Zarel) [inline commands.js]: Also consider that the reason we do `user.games` in the first place is that the user might be in the game but not the room (if they've disconnected, for instance).

Comment (sirDonovan) [inline commands.js]: Good point; updated.

**Pull request merged**